resources/views/backend/hotels/show.blade.php

```diff
@@ -0,0 +1,93 @@
+@extends('backend.layouts.main')
+@section('title', __('Hotels managment'))
+@section('content')
+<div class="content-wrapper">
+  <section class="content-header">
+    <h1>
+      {{ __('Hotels managment') }}
+      <small>
+        {{ __('Hotel detail') }}
+      </small>
+    </h1>
+    <ol class="breadcrumb">
+      <li><a href="#"><i class="fa fa-dashboard"></i>{{ __('Home') }}</a></li>
+      <li class="active">{{ __('Hotel detail') }}</li>
+    </ol>
+  </section>
+ <!-- Main content -->
+  <section class="content">
+    <div class="row">
+      <div class="col-md-5">
+        <div id="myCarousel" class="carousel slide" data-ride="carousel">
+          <!-- slides images hotel-->
+          <div class="carousel-inner">
+          {{-- show default image when has no image --}}
+            @if ($hotel->images->count() == 0)
+                <div class="item active">
+                  <img src="{{ asset('/images/default/default-thumbnail.jpg') }}">
```

Comment:
Could you please put this into `config/image.php`.
Name should be `default_thumbnail `.
Just use it like this here: `asset(config('image.default_thumbnail'))`.